documentation/versioned_docs/version-4.x/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@reveal-documentation/v4",
+  "version": "1.0.0",
+  "private": true,
+  "author": "Christopher Tannum <christopher.tannum@cognite.com>",
+  "scripts": {
+    "replace-4.x:copy-docs": "yarn copyfiles -u 3 -e \"../../docs/**/*.json\" -e \"../../docs/**/*.lock\" \"../../docs/**/*\" .",
+    "replace-4.x:update-site-imports": "yarn replace \"'@site/docs/\" \"'@site/versioned_docs/version-4.x/\" . --include '*.ts*,*.mdx' -r",
+    "replace-4.x:update-sdk": "yarn replace \"\\'@cognite/sdk\\'\" \"''@cognite/sdk-4.x''\" . --include '*.ts*' -r",
+    "replace-4.x:update-reveal": "yarn replace \"\\'@cognite/reveal\\'\" \"''@cognite/reveal-4.x''\" . --include '*.ts*' -r",
+    "replace-4.x:update-runnable": "yarn replace '^```(js[x]?) runnable$' '```$1 runnable-4x' . --include '*.mdx' -r",
+    "replace-4.x": "yarn add @types/three@0.146.0 && yarn replace-4.x:copy-docs && yarn replace-4.x:update-sdk && yarn replace-4.x:update-reveal && yarn replace-4.x:update-runnable && yarn replace-4.x:update-site-imports"
```

Comment:
Wtf is this????? 😂 (yarn add @types/three@0.146.0)

Adding a new dependency in the replace script!?!?!?!?!